src/MicrosoftTeams.schema.json

```diff
@@ -87,10 +87,11 @@
         },
         "accentColor": {
             "type": "string",
-            "description": "A color to use in conjunction with the app's icons. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
         },
         "galleryTabs": {
             "type": "array",
+            "description": "The set of tabs that users can optionally add to Team's as a presentation canvas.",
```

Comment:
Teams not Team's.   Also, "presentation canvas" is internal terminology.  Happy to brainstorm how to present this.